src/main/java/org/datacleaner/extension/elasticsearch/ElasticSearchTransformer.java

```diff
@@ -25,4 +25,5 @@
 
     public static final String PROPERTY_ES_DATASTORE = "ElasticSearch index";
     public static final String PROPERTY_DOCUMENT_TYPE = "Document type";
+    public static final String CONNECTION_TYPE_ERROR = "This component requires the connection to ElasticSearch to be done as a NODE or via TRANSPORT protocol";
```

Comment:
This is not really a part of the transformer's responsibility. I would rather create a new exception type with this text in it.